pkg/pusher/pusher.go

```diff
@@ -123,6 +134,11 @@ func (s *Service) setChunkAsSynced(ctx context.Context, addr swarm.Address) {
 		s.metrics.ErrorSettingChunkToSynced.Inc()
 	} else {
 		s.metrics.TotalChunksSynced.Inc()
+		ta, err := s.tag.GetByAddress(addr)
+		if err != nil {
+			s.logger.Errorf("pusher: error setting tag to synced: %v", err)
```

Comment:
This log line should be of Debug severity as it exposes the error.

This log line is not correct as the error is by GetByAddress, not setting the tag as synced (which is on the line below).

```suggestion
			s.logger.Debugf("pusher: get tag by address %s: %v", ch.Address(), err)
```